internal/sweep/sweep_test.go

```diff
@@ -1,9 +1,10 @@
 package sweep_test
 
 import (
-	"github.com/equinix/terraform-provider-equinix/internal/resources/metal/vlan"
 	"testing"
 
+	"github.com/equinix/terraform-provider-equinix/internal/resources/metal/vlan"
```

Comment:
We should get something in place to enforce that imports are declared in the "correct" order, for some agreed-upon definition of correct, to avoid stylistic churn like this.

Reply:
Agreed. I stalled on getting this PR consistent because there were some import changes affected but the tools I used (golangci-lint, goimports) didn't have a strong enough opinion to update the existing import blocks. Updating beyond the files affected would be preferrable in another PR or at least a separate commit.

goimports wouldn't regroup all imports ignoring existing arbitrary white-space. Since it honors the existing gaps the desired 3 group import stanzas (built-in, external, internal) was not achieved.